bq-client.js

```diff
@@ -20,7 +20,7 @@ module.exports = (projectName, dataSetName, refreshUrl)=>{
 
   var mod = {
     refreshToken,
-    insert(tableName, data, nowDate) {
+    insert(tableName, data, templateSuffix, nowDate) {
```

Comment:
@rodrigopavezi I think you should keep the same order `tableName, data, nowDate` otherwise any clients calling insert with all three parameters will end up supplying a templateSuffix that is actually a date.